Source/WebKit/WebProcess/Extensions/API/Cocoa/WebExtensionAPINamespaceCocoa.mm

```diff
@@ -266,6 +273,18 @@
 }
 #endif
 
+#if ENABLE(WK_WEB_EXTENSIONS_BOOKMARKS)
+WebExtensionAPIBookmarks& WebExtensionAPINamespace::bookmarks()
+{
+    // Documentation: https://developer.chrome.com/docs/extensions/reference/api/bookmarks
```

Comment:
we use mdn for documentation links
https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/bookmarks